package.json

```
@@ -17,20 +21,16 @@
     "eslint-import-resolver-webpack": "0.11.1",
     "eslint-plugin-flowtype": "3.12.2",
     "eslint-plugin-import": "2.18.2",
+    "eslint-plugin-pmedianetwork": "file:.",
     "eslint-plugin-jsx-a11y": "6.2.3",
     "eslint-plugin-no-only-tests": "2.3.1",
     "eslint-plugin-prettier": "3.1.2",
     "eslint-plugin-react": "7.16.0",
     "eslint-plugin-react-hooks": "2.2.0"
   },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/pmedianetwork/eslint-config.git"
+  "devDependencies": {
+    "jest": "25.1.0",
+    "prettier": "^1.19.1"
```

another ^ snuck in. maybe we need an eslint rule for that, too 😄 

```suggestion
    "prettier": "1.19.1"
```